hazelcast/src/test/java/com/hazelcast/internal/jmx/MBeanDataHolder.java

```diff
@@ -56,6 +56,13 @@ public MBeanDataHolder(TestHazelcastInstanceFactory factory) {
         mbs = ManagementFactory.getPlatformMBeanServer();
     }
 
+    public MBeanDataHolder(TestHazelcastInstanceFactory factory, Config config) {
```

Comment:
can you refactor `MBeanDataHolder(TestHazelcastInstanceFactory)` to use this method also?

Reply:
Done